content/applications/sales/subscriptions/configuration/subscription_products.rst

```diff
@@ -1,54 +1,74 @@
-follow these steps:
+By integrating closely with the Odoo *Sales* app, the *Subscriptions* app enables users to sell
+subscription products alongside regular sales products. While regular products are sold on a
+one-time basis, subscription products are sold on a renewing basis, generating recurring revenue.
 
-1. **Create your own subscription templates**
-2. **Create your own subscription products with the right settings**
+Configure recurrence periods
+============================
 
-As a result, you will be able to manage your subscriptions like any other product, create your
-quotations and continue the sales flow to track the number of subscriptions you sell and manage the
-revenue they generate.
+To get started with subscriptions, first make sure that the *recurrence periods* are configured as
+needed.
 
-Configuration
-=============
+Recurrence periods are the time periods in which subscriptions renew. They designate how often
+the customer pays for (and receives) subscription products.
 
-Go to :menuselection:`Subscriptions --> Subscription products` to learn how you can configure your
-own subscription products. You have the possibility to create a new product or edit an existing one.
-Once named, be careful to select the option *Can be sold* and deselect *Can be purchased*. For the
-product type, it is recommended to use *Service* for subscription products as they are non-material
-products that you provide to your customers. Finally, you can adapt your prices and also add an
-internal reference.
+To configure recurrence periods, go to :menuselection:`Subscriptions --> Configuration -->
+Recurrence Periods`.
 
-.. image:: subscription_products/subscription-products-configuration.png
-  :align: center
-  :alt: View of a subscription product form in Odoo Subscriptions
+The *Subscriptions* app comes with some basic recurrence periods already configured. These can be
+edited as needed, and any number of new recurrence periods can be added.
+
+To create a new recurrence period, click :guilabel:`New`. Then, type in the :guilabel:`Name` and
+:guilabel:`Duration` of the recurrence period, and select the :guilabel:`Unit` that defines the
+duration.
+
+.. example::
+   To create a recurrence period such that a subscription renews every two weeks, set the
+   :guilabel:`Duration` to `2` and the :guilabel:`Unit` to `Weeks`.
+
+   .. image:: subscription_products/recurrence-period-form.png
+      :align: center
+      :alt: A recurrence period of 2 weeks.
+
+Configure the product form
+==========================
+
+To create a new subscription product, navigate to the *Subscriptions* app. Then go to
+:menuselection:`Subscriptions --> Products`, and click :guilabel:`New` to create a new product.
+
+The :guilabel:`Product Type` for the new product is automatically set to :guilabel:`Service`.
+Subscription products can be set to other types as well; however, they currently *cannot* be set to
+:guilabel:`Storable Product`.
```

Comment:
I would include an explanation for why subscription products can't be configured with the storable product type.

Reply:
The explanation is just that the developers haven't deployed this feature yet :) they are planning to add it for v17. So I kept this part the same.